cypress/e2e/hub/signature-keys.cy.ts

```diff
@@ -2,16 +2,59 @@ import { SignatureKeys } from './constants';
 import { pulpAPI } from '../../support/formatApiPathForHub';
 
 describe('Signature Keys', () => {
-  before(() => {
+  beforeEach(() => {
```

Comment:
`before` should do that job here, since we are leveraging the `cy.sesssion` to avoid to login every single test, right? Did I miss something else?

Reply:
yes, thanks for bringing it up Kersom
with before hook, tests fail in this case, only with beforeEach hook the two tests I wrote pass.